docs/guides/4-custom-rulesets.md

```diff
@@ -21,10 +25,13 @@ Spectral has [built-in functions](../reference/functions.md) such as `truthy` or
 
 ### Given
 
-The `given` property is one of only two on required properties on each rule definition (the other being `then`).
+The `given` property is conceptually quite like a selector in CSS, in that it picks the part of the document to apply rules to.
+
+It has a specific syntax known as [JSON Path](https://jsonpath.com/), which if you are familiar with XPath is quite similar. JSON Path is not yet a standard (it [will be](https://tools.ietf.org/html/draft-normington-jsonpath-00) someday), and has a few competing implementations. Spectral uses [jsonpath-plus](https://www.npmjs.com/package/jsonpath-plus) for the implementation, which supports all the main JSON Path functionality and a little bit more.
+
+Your `given` value can be a string containing any valid JSON Path Plus expression, or an array of expressions to apply a rule to multiple parts of a document.
```

Comment:
```suggestion
Your `given` value can be a string containing any valid JSON Path expression, or an array of expressions to apply a rule to multiple parts of a document.
```

Reply:
This is intentional because we're using JSON Path Plus, so if it's valid JSON Path Plus it's good to use here. Unless we're trying to hide the plus stuff in the hopes that nobody uses it?